client/components/admin_dashboard/ReservationTable.jsx

```diff
@@ -0,0 +1,12 @@
+import Box from "@mui/material/Box";
+import { fetchAppointments } from "@/actions/form";
+import Grid from "./Grid";
+
+export default async function ReservationTable() {
+  const rows = await fetchAppointments();
+  return (
+    <Box sx={{ height: 400, width: "100%" }}>
```

Comment:
Is there a reason to set an explicit height and width? FYI this would probably not be the most maintainable thing to do in the real world. Using normal block or flex layout would probably still give you what you want